src/infra/main/template.yaml

```diff
@@ -169,7 +169,7 @@ Resources:
     Properties:
       FunctionName: !Sub "${AWS::StackName}-InterventionsProcessorFunction"
       Description: "Processes intervention events and updates account's intervention status."
-      CodeUri: ../../../dist/interventions-processor-handler
+      CodeUri: ../../handlers
```

Comment:
wondering if this can be configured once in global config then does not have to be done per lambda

Reply:
done - also moved kmsKeyArn as they all point to the same key